NAudio/Midi/MidiIn.cs

```diff
@@ -48,6 +57,16 @@ public MidiIn(int deviceNo)
         /// </summary>
         public void Close() 
         {
+            //  Free up all created and allocated buffers for incoming Sysex messages

```

Comment:
this code should move into the `Dispose` method so that it gets cleared up if you use it with a `using` block

Reply:
I agree - I had just re-read your Midi doc, which says to use Dispose(), rather than Close() and thought that. I never even considered the issue of "using". Would you just do that or to I need to update the PR in some way? Also, have I made the change on the wrong branch? I notice that my ChannelAftertouch fix went on the V2 branch. The use of branches isn't very clear I'm afraid.

Reply:
You can push new changes to your branch and this PR will automatically update itself.
NAudio has had just one branch for years, but the v2 stuff is being done in a separate branch and is going to be imminently released (when I have time to fix). I've been merging changes from v1 to v2 in the meantime.

Reply:
OK - new commit pushed, moving the cleanup code to Dispose(). I see I have a lot to learn about Git before I attempt any more contributions to public open source projects!!